src/printer.cpp

```diff
@@ -529,6 +529,7 @@ std::string Printer::printModel(const ModelPtr &model, bool autoIds) const
 
     // Generate a pretty-print version of the model using libxml2.
     XmlDocPtr xmlDoc = std::make_shared<XmlDoc>();
+    xmlKeepBlanksDefault(0); // see: http://www.xmlsoft.org/html/libxml-tree.html#xmlDocDumpFormatMemoryEnc
```

Comment:
Don't need the comment. If anything, I would expect further explanation, in your comment at line 530, as why we need to call this method.

Reply:
Done in b43279a